### ISSUE TITLE
 error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by CosbyBs86** [innovation-system/electron-kiosk#19]
Hello all
i'm trying to use AppImage file in a linux system and I got this error: " error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory". is there any way to have a self-contained package?

**Comment by robertsLando** [comment]
Try to `sudo apt install libnss3`

**Comment by CosbyBs86** [comment]
Yes this for sure is a solution but I don't want to install anything on the target host. I would like to have a self-contained package. Anybody can help me?
